requirements.txt

```diff
@@ -2,6 +2,7 @@ cryptography
 cbor2
 ecdsa
 attrs
+certvalidator
```

Comment:
It looks like a well-maintained project so probably we can go ahead with using it (at least for now). I'm just wondering if it would be possible to do the certificate validation with the [ssl library](https://docs.python.org/3/library/ssl.html) that's part of the Python standard library?

The only issue that is see is that in the long run we probably want to support [CBOR certificates](https://tools.ietf.org/id/draft-mattsson-cose-cbor-cert-compress-04.html) too since they will be heavily used by lightweight protocol such as EDHOC and OSCORE, which in turn rely on COSE. I'm not sure if the `certvalidator` package will ever support CBOR certificates.

Reply:
The good news related to "CBOR Encoded X.509 Certificates" draft is that one mode truly is an on-the-wire encoding change and not a data content change; meaning that cert fingerprints, signatures, validation, etc. all apply to the canonical ASN.1 form and don't care that the transport may have been compressed in CBOR.

The other C509 mode uses native fingerprint and signatures and requires validator awareness.